sass/app.scss

```diff
@@ -42,7 +56,9 @@
 	display: inline-block;
 	max-width: 55%;
 	text-align: center;
+	@include borders(1px solid red, 1px solid blue);
 }
 .footer {
 	margin-top: 30px;
+	@include bgc(#eee);
```

Comment:
Interesting but would you create a mixin for such a thing, if code length is the same as property.
Probably we need more complex mixin for background
Let's try in future solve issue with background images for banners
background-attachment || background-color || background-image || background-position || background-repeat

So we will be able to set any of this property or leave it as default